### DISCUSSION TITLE
wazuh app in so-kibana

### DISCUSSION THREAD

**Discussion by Alexsidin** [Security-Onion-Solutions/securityonion#3433] [Unsupported Versions] [answered]
Hello, I'm new to security onion, please tell me how to integrate the wazuh application into the kibana web panel in security-onion 2.3.30. Thank.

**Comment by launchdaemon** [comment]
If you click the Dashboard link at the top there are a number of pre-defined ones including a "Security Onion - Wazuh" dashboard.  Is that what you had in mind? If you don't see any results initially make sure you've set an appropriate time period to search for in the top right.

**Comment by weslambert** [comment, marked as answer]
We have considered supporting the Wazuh app for Kibana in the future, but do not currently support it.

**Reply by b4b857f6ee** [reply]
Yes support it <3
